payments/tests/test_notifications.py

```diff
@@ -95,6 +107,23 @@ def test_reservation_created_notification(order_with_products):
     )
 
 
+@pytest.mark.django_db
+@override_settings(RESPA_MAILS_ENABLED=True)
+def test_paid_reservation_approved_notification(requested_reservation_with_order):
+    res = requested_reservation_with_order
+    user = res.user
+    user.preferred_language = 'fi'
+    user.save()
+
+    res.set_state(Reservation.WAITING_FOR_PAYMENT, user)
+
+    assert len(mail.outbox) == 1
```

Comment:
There's also a `mailoutbox` fixture with pytest django: not sure if easier to use than importing `mail.outbox` separately.

Reply:
Nice, didn't know about that. Let me try it.

Reply:
Changed. That's quite handy.